data/styles/clear/include/Subways.mapcss

```diff
@@ -856,13 +865,6 @@ node|z15-16[railway=station][transport=subway][city=valencia] {icon-image: subwa
 node|z17-[railway=station][transport=subway][city=valencia] {icon-image: subway-valencia-m.svg;text-optional: true;}
 node|z17-[railway=subway_entrance][city=valencia] {icon-image: subway-valencia-s.svg;text-optional: true;}
 
-/* Valencia Subway Station */
```

Comment:
Why is it removed?

Reply:
It's a duplicate, there's already drawing rules for Valencia L852-L857